bundle/edu.gemini.util.skycalc/src/main/java/edu/gemini/skycalc/TwilightBoundType.java

```diff
@@ -15,8 +15,12 @@
 
     private static final long serialVersionUID = 1;
 
+    // Horizon geometric correction from p. 24 of the Skycalc manual: sqrt(2 * elevation / Re) (radians)
+    static double Re = 6378140; // Radius of the Earth (meters)
+    static double SiteElevation = 4213; // Mauna Kea elevation (meters)
```

Comment:
Is there already an altittude on `Site`?
Possibly we should use `squants` distances in meters rather than `double`

Reply:
Earth radius is already defined on `ImprovedSkyCalcMethods` and yes, elevations are defined on `Site`.